docs/modules/ROOT/pages/apdx-operatorChoice.adoc

```diff
@@ -1,111 +1,111 @@
 [[which-operator]]
-= Which operator do I need?
+== Which operator do I need?
 
-TIP: In this section, if an operator is specific to https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html[Flux] or https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html[Mono], it is prefixed and linked accordingly, like this: https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html#fromArray-T:A-[Flux#fromArray]. Common operators have no prefix, and links to both implementations are provided, for example: `just` (https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html#just-T%2E%2E%2E-[Flux]|link:https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html#just-T-[Mono]). When a specific use case is covered by a combination of operators, it is presented as a method call, with a leading dot and parameters in parentheses, as follows: `.methodCall(parameter)`.
+TIP: In this section, if an operator is specific to {javadoc}/reactor/core/publisher/Flux.html[Flux] or {javadoc}/reactor/core/publisher/Mono.html[Mono], it is prefixed and linked accordingly, like this: {javadoc}/reactor/core/publisher/Flux.html#fromArray-T:A-[Flux#fromArray]. Common operators have no prefix, and links to both implementations are provided, for example: `just` ({javadoc}/reactor/core/publisher/Flux.html#just-T%2E%2E%2E-[Flux]|link:{javadoc}/reactor/core/publisher/Mono.html#just-T-[Mono]). When a specific use case is covered by a combination of operators, it is presented as a method call, with a leading dot and parameters in parentheses, as follows: `.methodCall(parameter)`.
 
 //TODO flux:  publishOn/subscribeOn/cancelOn
 //transformDeferred/transform, repeatWhen, sort, startWith
 //TODO Mono.sequenceEqual
 
 I want to deal with:
 
-* <<which.create>>
+* xref:apdx-operatorChoice.adoc#which.create[Creating a New Sequence...]
 
-* <<which.values>>
+* xref:apdx-operatorChoice.adoc#which.values[Transforming an Existing Sequence]
 
-* <<which.filtering>>
+* xref:apdx-operatorChoice.adoc#which.filtering[Filtering a Sequence]
 
-* <<which.peeking>>
+* xref:apdx-operatorChoice.adoc#which.peeking[Peeking into a Sequence]
 
-* <<which.errors>>
+* xref:apdx-operatorChoice.adoc#which.errors[Handling Errors]
 
-* <<which.time>>
+* xref:apdx-operatorChoice.adoc#which.time[Working with Time]
 
-* <<which.window>>
+* xref:apdx-operatorChoice.adoc#which.window[Splitting a {javadoc}/reactor/core/publisher/Flux.html[Flux]]
```

Comment:
I suggest not linking Flux in the link to a header, it's not great UX if you open the javadoc if you point to the Flux word by mistake.

Reply:
Fixed in https://github.com/reactor/reactor-core/pull/3806/commits/9243ea5bd2f274cf4e514e064ad88014f0c09afb